hugegraph-server/hugegraph-dist/src/assembly/travis/run-api-test.sh

```
@@ -94,17 +136,22 @@ authentication: {
 }" >> $GREMLIN_SERVER_CONF
 
 # start server
+SERVER_START_ATTEMPTED=true
 $TRAVIS_DIR/start-server.sh $SERVER_DIR $BACKEND $JACOCO_PORT || (cat $SERVER_DIR/logs/hugegraph-server.log && exit 1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)